resources/views/account/accept-eula.blade.php

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 Boilerplate</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+
+    @if ($signature)
+        <center>
+            <img src="{{ $logo }}">
+        </center>
+    @endif
+    <br>
```

Comment:
This bis some jank-ass HTML, but was meant for placement only :) `<center></center>` has been deprecated for quite some time.